pkg/scheduler/framework/plugins/volumebinding/binder.go

```diff
@@ -503,11 +503,14 @@ func (b *volumeBinder) bindAPIUpdate(pod *v1.Pod, bindings []*BindingInfo, claim
 		klog.V(5).InfoS("bindAPIUpdate: binding PV to PVC", "pod", klog.KObj(pod), "PV", klog.KObj(binding.pv), "PVC", klog.KObj(binding.pvc))
 		// TODO: does it hurt if we make an api call and nothing needs to be updated?
 		klog.V(2).InfoS("Claim bound to volume", "PVC", klog.KObj(binding.pvc), "PV", klog.KObj(binding.pv))
+
+		// Note: DO NOT use `RetryOnConflict`, it's safe to do the schedule again on update conflict
```

Comment:
Are these notes really needed..?

Reply:
Maybe someone else will encounter the same issue, then try to modify the update conflict here again.

Reply:
OK, updated.